R/parameters.R

```diff
@@ -19,18 +19,41 @@ get_parameters <- function(iso3c = NULL,
     contact_matrix_set <- squire::get_mixing_matrix(iso3c = iso3c)
   }
 
-  c(
-    squire::parameters_explicit_SEEIR(
+  prob_asymp <- c(0.3, 0.3, rep(0.2, 15))
+  IAsymp_0 <- c(rep(0L, 17))
+  dur_IAsymp <- 2.1
+
+  parameters <- list(
+    sq = squire::parameters_explicit_SEEIR(
       population = population,
       country = get_country(iso3c),
       contact_matrix_set = contact_matrix_set,
-      dt = 1, # dt should always be 1 as individual is always discrete time
+      dt = 1, # dt should always be 1 as individual is always a discrete time
       time_period = time_period,
       ...
     ),
+    prob_asymp = prob_asymp,
```

Comment:
here you are you just rewriting this latter with newlist etc. Would recommend simplifying